### ISSUE TITLE
Default root folder

### ISSUE THREAD

**Issue by jwueller** [pulsejet/memories#1186]
**Is your feature request related to a problem? Please describe.**
I am configuring this app for users of varying affinity for technology, and many of them get stuck on the root folder selection because they don't understand what it does.

**Describe the solution you'd like**
Allow skipping the initial setup step by configuring a sensible default, like the user's home folder. I would rather have people exclude things after the fact rather than not being able to find what they are looking for. This is how the stock Photos app behaves too.

**Describe alternatives you've considered**
If there has to be an initial question for every user, I would love to see a trivial setup option in addition to manual folder selection. For example, "Include everything" and "Select specific folders".

**Additional information**
I can attempt a PR if desired.

**Comment by vbelloir** [comment]
Hi, 
what do you mean by "get stuck on the root selection".
I think I'm facing the same issue.
I use memories in family context, but for a non understanding reason, I managed to finish the root folder selection and use the android app, but for some others, I choose a folder, but the app get me back to the root folder selection, again and again... 
Is that what you mean?

**Comment by jwueller** [comment]
No, I'm saying people don't understand what that dialog is asking of them, so they just never get past it because they are afraid to break something. Or they will keep asking me for help.

**Comment by vbelloir** [comment]
ok, understand. 
But I agree with you, user should be able to skeep this setting, and configure later in app.

**Comment by primelionzard** [comment]
i have this root folder selection issues on firsr install

**Comment by jwueller** [comment]
@pulsejet Apologies for the ping. Would you accept a PR for this? I don't want to waste everyone's time by writing and reviewing code that isn't going to get merged due to design decisions.

**Comment by primelionzard** [comment]
An exception oddb.oc_memories' doesn't exist in engine 'nextclouddb.oc_memories' doesn't exist in engine 'nextclouddb.oc_memories' doesn't exist in engine

I get this in mine when it fails and send me back to page to select a root folder
